.nvmrc

```diff
@@ -0,0 +1 @@
+v12.16.1
```

Comment:
Why to include this? There are plenty on dot files/configs. For example I use brew for node.

Reply:
For install microbundle next version, is needed a node version > than 10.
So i use this file for switch node version with nvm

Reply:
But I don't use nvm. Why include it in the repo?